.idea/codeStyles/Project.xml

```diff
@@ -328,4 +328,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
Can you revert these changes as well? 

I will add `insert_final_newline = true` to `.editorconfig` so that this doesn't happen again.

Reply:
ignored the completed local changes in .idea folder